lib/src/react/render/render.dart

```diff
@@ -109,6 +109,7 @@ RenderResult render(
   Node? container,
   Node? baseElement,
   bool hydrate = false,
+  bool legacyRoot = true,
```

Comment:
Could we add legacyRoot to the doc comment? 

Could go based off the site's docs: https://testing-library.com/docs/react-testing-library/api/#legacyroot

(Though reminder our default value is different from theirs)